spec/controllers/hub/bulk_actions/change_assignee_and_status_controller_spec.rb

```diff
@@ -30,6 +30,19 @@
         expect(assigns(:current_tr_statuses)).not_to include unselected_tax_return.current_state
       end
 
+      context "preserving order of tax_return creation dates" do
+        before do
+          tax_return_1.update(created_at: tax_return_3.created_at + 5.minutes)
+        end
+
+        it "makes sure the tr_statuses are sorted by tax_return created_at dates" do
+          get :edit, params: params
+
+          expect(assigns(:current_tr_statuses)).to eq ["review_signature_requested", "file_ready_to_file"]
+          expect(assigns(:current_tr_statuses)).not_to include unselected_tax_return.current_state
+        end
+      end
```

Comment:
note that the test above it tests whether it `match_array` which doesnt care about order

Reply:
where do you use match_array? 🤔 i think line 42 cares about order but i might be wrong?

Reply:
The existing `match_array` spec already exists on line 29. I added the new spec (line 42) which does check for the order. I _can_ change the original spec to order by tax return creation dates there too...should I? @anisharamnani